csharp/src/Microsoft.ML.OnnxRuntime/GlobalMethods.cs

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Microsoft.ML.OnnxRuntime
+{
+    public static class GlobalMethods
```

Comment:
>ublic static class GlobalMethods [](start = 5, length = 32)

I would recommend to expose this through the Environment class? As Adam says to be consistent with Java?

Reply:
Done